web/generate_site/templates/head.html

```diff
@@ -2,7 +2,12 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>PyLadiesCon 2023</title>
+    <meta property="og:title" content="{{ og_title }}">
+    <meta property="og:description" content="{{ og_description }}">
+    <meta property="og:type" content="{{ og_type }}">
+    <meta property="og:url" content="{{ og_url }}">
+    <meta property="og:image" content="{{ og_image_url }}">
```

Comment:
```suggestion
    <meta property="og:image" content="{{ og_image_url }}">
    <meta property="og:image:alt" content="{{ og_image_alt }}">
```

Ref: https://ogp.me/#structured